src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/custom.py

```diff
@@ -43,50 +52,55 @@ def _get_deserializer():
     return Deserializer(client_models)
 
 
-def get_environment_variable_or_die(name):
-    """Returns the value of an environment variable with given name or raises CLIError exception.
-
-    :param str name: name of the environment variable.
-    :raise CLIError: if environment variable not found or empty.
-    """
-    value = os.environ.get(name, None)
-    if not value:
-        raise CLIError('Please set {0} environment variable.'.format(name))
-    return value
-
-
 def update_cluster_create_parameters_with_env_variables(params):
     """Replaces placeholders with information from the environment variables.
 
     Currently we support replacing of storage account name and key in mount volumes.
 
     :param models.ClusterCreateParameters params: cluster creation parameters to patch.
     """
+    storage_account_name = az_config.get('batchai', 'storage_account',
+                                         fallback=AZURE_BATCHAI_STORAGE_ACCOUNT_PLACEHOLDER)
+    storage_account_key = az_config.get('batchai', 'storage_key', fallback=AZURE_BATCHAI_STORAGE_KEY_PLACEHOLDER)
```

Comment:
Do you need the placeholder?
Why not `fallback=None` like below.

Reply:
it makes the code simpler and logic cleaner I think. if there are no config values I am initializing the variables with place holders and do not check if those values are really needed until the very end. if after all manipulations there are any placeholders left, the code reports an error.

Reply:
in the case below, I do know that those config values are needed, so I am reporting the error immediately